includes/backupMappings.js

```diff
@@ -0,0 +1,260 @@
+// Copyright © 2017, 2023 IBM Corp. All rights reserved.
```

Comment:
Here too. I'm not sure if we want year when actual file was created or when we merge it in `main`, but we do 2017, 2024 elsewhere